js/menu.js

```diff
@@ -1,18 +1,31 @@
-$('.show-menu-button').on('click', function() {
+function showMenu() {
 	$('.ui.vertical.sidebar').addClass('overlay visible');
+}
+
+function hideMenu() {
```

Comment:
@fatimarafiqui Block comments are required as part of every PR. Please add them **before** you submit PR since this feedback has come up a couple of times in the last few PRs.